.credo.exs

```diff
@@ -129,7 +129,7 @@
         {Credo.Check.Refactor.MatchInCondition, []},
         {Credo.Check.Refactor.NegatedConditionsInUnless, []},
         {Credo.Check.Refactor.NegatedConditionsWithElse, []},
-        {Credo.Check.Refactor.Nesting, []},
+        {Credo.Check.Refactor.Nesting, [max_nesting: 3]},
```

Comment:
A little bit more power?

Reply:
Why do we need this?

Reply:
We need more power in writing code :D I belive that 3 leveles of nesting is acceptable. Some functions in ConfigReader uses this